rev_list.go

```diff
@@ -0,0 +1,115 @@
+package git
+
+import (
+	"io"
+
+	"gopkg.in/src-d/go-git.v4/plumbing"
+)
+
+// RevListObjects gets all the hashes from all the reachable objects from the
+// given commits. Ignore hashes are objects that you don't want back into the
+// result. All that objects must be accessible from the Repository.
```

Comment:
I really don't understand the phrase "Ignore hashes are objects that you don't want back into the result"